packages/similarity/src/cosine-similarity.js

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) AXA Group Operations Spain S.A.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining
+ * a copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sublicense, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be
+ * included in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+ * EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+ * NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+ * LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+ * WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+const { NlpUtil } = require('../../node-nlp/src/nlp');
```

Comment:
One of the reasons of splititing into packages the v4 was to reduce the size and make it proper for web.
With this dependency, this is broken, because all the packages and languages will be included in similarity.
This is the main reason of having an IoC container, so the dependencies can be registered when needed, example, the languages. 
So this dependency must be avoided in @nlpjs/similarity

Reply:
Ok, I see your point. I've updated the code, removing all external dependencies, in this way will work by default with latin characters. To make it work with oriental locales, you'll need to install the locale package + basic package.
I've put examples of both cases in description.

P.S: I had to remove oriental test cases to keep it independent. If you want also oriental tests, I can add locales as devDependencies in that package.